README.md

```diff
@@ -25,3 +25,84 @@ are up to date.
 
 Carson will never read the contents of a pull requests, ie it never parses the code
 to check for typos or other automated fixes.
+
+## Features
+
+Here is all the things Carson can help you with. Not all features are enabled. A
+list of enabled features for a specific repository is defined in `config/services.yaml`.
+
+### Automatic labeling from content
+
+Using the issue template, carson can figure out to add labels "Bug", "Feature",
+"BC Break" and "Deprecation" on an issue and pull request.
+
+### Add "Needs Review" label on pull requests
+
+When a PR is opened, then Carson will add "Needs Review".
+
+### Add "Needs Review" label on bugs issues
+
+If an issue is labeled with "Bug", then Carson will add "Needs Review".
+
+### Update pull request title with component labels
+
+When a PR is labeled, we will look for component labels, ie labels with color #dddddd.
+These labels' names will be added to the PR title.
```

Comment:
```suggestion
When a PR is labeled, Carson looks for component labels (i.e. labels with color #dddddd).
These label's names will be added to the PR title (e.g. `[HttpKernel]`).
```

Reply:
I think it should be `These labels' names`. I'm referring to many `labels` and if they are owning something I should add the `'s`. But writing `labels's` is something that should be avoided. One will short it to `labels'`.

Reply:
I updated your comment to get the other changes.